openbas-api/src/main/java/io/openbas/rest/tag/TagApi.java

```diff
@@ -26,6 +31,10 @@
 import org.springframework.web.bind.annotation.*;
 
 @RestController
+@io.swagger.v3.oas.annotations.tags.Tag(
```

Comment:
question: tags in @operation create new sections in the swagger. According to the doc, the behavior for this @Tag should be the same. Is that the case and if yes, is that what we want?

Reply:
Yes, all the endpoints in TagApi go to the same section so it's simpler to apply it on the class rather than on all endpoints. If not all the endpoint would go to the same section, we would add it at the endpoint level instead of the class.